mesonbuild/modules/windows.py

```diff
@@ -66,6 +67,7 @@ def _find_resource_compiler(self, state):
         for (arg, match, rc_type) in [
                 ('/?', '^.*Microsoft.*Resource Compiler.*$', ResourceCompilerType.rc),
                 ('--version', '^.*GNU windres.*$', ResourceCompilerType.windres),
+                ('--version', '^.*Wine Resource Compiler.*$', ResourceCompilerType.wrc),
```

Comment:
It's named "wrc." Yes, it needs to be in a cross file because:

- winelib builds are always cross builds.
- The spirit of winelib is that you "just" replace the compiler/linker with winegcc and windres with wrc, so being able to specify "windres = 'wrc'" in the crossfile makes sense.
- wrc needs to be paired with winegcc, so we wouldn't want anyone to accidentally get wrc instead of windres if they for whatever reason forgot to install windres. This needs to be opt-in.

If/when we add a "winelib" platform like cygwin, then we might want to then search for wrc. Until then I don't think it makes sense though.